### DISCUSSION TITLE
[android] Investigate FABs

### DISCUSSION THREAD

**Discussion by hawkrives** [StoDevX/AAO-React-Native#5553] [Ideas]
> inspired by discussion in #3182 

Android uses FABs (floating action buttons) to draw the eye / finger to certain "primary actions" on a screen.

I'd like to use this issue to discuss what views should use FABs, and which actions should be promoted to FABiness.

We should maintain a checklist in this OP with the current decisions.

---

- [ ] Student Work: "Open Posting", as a centered, full-width FAB
- [x] Directory detail view: "Call" and "Email" via  #3558

**Comment by rye** [comment]
`react-native-paper` was one thing that has FABs nicely-implemented.

**Comment by drewvolz** [comment]
We've implemented a FAB in the android app but we could still track where we're using it and more use cases for it.
